### PR TITLE
add Gradient Descent reference in tutorial

### PR DESCRIPTION
**Pull request by fayeshine**
Add a wiki reference for Gradient Descent in TF mechanics 101

### CODE REVIEW

**Comment by caisq** [comment]
@tensorflow-jenkins , test this please.
